Added a portion to FAQ to include modifying context object (#4861)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/react-native/index.md

    }
  }
```
### How do I add to the context Object?

> Similar to the integrations object above you'll need to use a plugin to access and modify the context object. For example, if you'd like to add `context.groupId` to every Track call this can be added like so:

Reviewer: ```suggestion
Like with the integrations object above, you'll need to use a plugin to access and modify the context object. For example, if you'd like to add `context.groupId` to every Track call, you should create a plugin like this:
```
